packages/theme/src/steps/responsive.less

```
@@ -0,0 +1,36 @@
+@import '../custom.less';
+@import './vars.less';
+
+@steps-prefix-cls: ~'@{css-prefix}steps';
+@steps-line-prefix-cls: ~'@{css-prefix}steps-line';
+@steps-standard-prefix-cls: ~'@{css-prefix}steps-standard';
+
+@media screen and (max-width: 1280px) {
+  .@{steps-line-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word steps-line-prefix-cls (CssSyntaxError)

(CssSyntaxError)